tinymod/plugins/code_metrics.py

```diff
@@ -0,0 +1,209 @@
+import math
+import shelve
+from typing import Annotated, Optional
+import hashlib
+import tempfile
+from asyncio.subprocess import PIPE
```

Comment:
The async executor I'm using isn't actually `asyncio` but rather [`scarletio`](https://github.com/HuyaneMatsu/scarletio).